guide/src/advanced/features.md

```diff
@@ -118,6 +118,7 @@ __Feature comparison between Hermes and the Go relayer__
 | Packet_TimeoutClose_P  | ✅    | ✅     |
 | Packet_Optimistic      | ❌    | ❌     | relay packets over non-Open channels
 |                        |       |        |
+| Modular Architecture   | ❌    | ✅     | defined interface can be implemented for different chain types
```

Comment:
Hermes features a `ChainEndpoint` trait which provides an interface to implement for different chain types. Penumbra is making use of this in their fork of Hermes already. That said the interface is far from perfect, and we expect to make substantial improvements to it when upstreaming the changes from the Penumbra fork.

Moreover, we are working on a much better and modular architecture in the [Hermes SDK](https://github.com/informalsystems/hermes-sdk) project, which will become the basis for Hermes v2.

```suggestion
| Modular Architecture   | ✅🚧  | ✅     | defined interface can be implemented for different chain types (basic interface exists in Hermes, but is being heavily improved on in the [Hermes SDK](https://github.com/informalsystems/hermes-sdk) project)
```